sklearn/metrics/ranking.py

```diff
@@ -26,13 +26,11 @@
 
 from ..utils import assert_all_finite
 from ..utils import check_consistent_length
-from ..utils import column_or_1d, check_array, check_X_y
+from ..utils import column_or_1d, check_array
 from ..utils.multiclass import type_of_target
 from ..utils.extmath import stable_cumsum
 from ..utils.sparsefuncs import count_nonzero
 from ..exceptions import UndefinedMetricWarning
-from ..preprocessing import label_binarize
```

Comment:
Seems that we need label_binarize here, it's now used in roc_auc_score for supporting binary y_true other than {0, 1} / {-1, 1}.